### ISSUE TITLE
Arrears Listing does not give correct balance

### ISSUE THREAD

**Issue by MargaretLangley** [BCS-io/letting#225]
Go to Arrears submenu
Click on Arrears List
Search for arrears of over 1000
7055 has a balance of 3000
Click on 7055 
Balance is actually 3750

**Comment by MargaretLangley** [comment]
Using a search for arrears over £50
                                   Account       Listed Value      Actual Value
Correct Value             6904             75.00                75.00
                                   6484            170.38               642.88      170.36 is the first line of accounts 
                                   6859            110.10               297.60      110.10 actually 2nd in list  
                                   6494            765.50               1,255.00   765.50 part way down account list
                                   7503            125.00               1250.00     No value in accounts of 125.00

**Comment by RichardWigley** [comment]
Account.balance_all was summing over unique rows and not all rows 

UNION

````
  id  | property_id | amount  
------+-------------+---------
 1344 |        1344 |       0
 1344 |        1344 | 2625.00
 1344 |        1344 |  375.00
(3 rows)
````

UNION ALL
````
  id  | property_id | amount  
------+-------------+---------
 1344 |        1344 |       0
 1344 |        1344 | 2625.00
 1344 |        1344 |  375.00
 1344 |        1344 |  375.00
 1344 |        1344 |  375.00
(5 rows)

````

FIX efa3252de036247bd0fa72c5d7e548d908d206ac

**Comment by MargaretLangley** [comment]
Tested, works  fine
